rct_optimizations/test/extrinsic_multi_static_camera_utest.cpp

```diff
@@ -1,24 +1,12 @@
 #include <gtest/gtest.h>
 #include <rct_optimizations/extrinsic_multi_static_camera.h>
 #include <rct_optimizations/ceres_math_utilities.h>
+#include <rct_optimizations_tests/utilities.h>
+#include <rct_optimizations_tests/observation_creator.h>
 
-void makePoints(std::size_t rows, std::size_t cols, double spacing, std::vector<Eigen::Vector3d>& points)
```

Comment:
Did this get replaced by the functions in `observation_creator.h`?

Reply:
Yes and no. This function duplicates an existing `Target` struct in `utilities.h`, so I just replaced it with a `Target` object